slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/InsertTest.scala

```diff
@@ -136,6 +137,28 @@ class InsertTest extends AsyncTest[JdbcTestDB] {
     } yield ()
   }
 
+  def testInsertOrUpdatePlainWithFuncDefinedPK: DBIOAction[Unit, _, _] = {
+    //FIXME remove this after fixed checkInsert issue
+    if (tdb.profile.isInstanceOf[DerbyProfile]) return DBIO.successful(())
```

Comment:
Well, we can't use `ifCap(jcap.insertOrUpdate)` here, since it will exclude some profiles, like `PostgresProfile`, `OracleProfile`, `SqliteProfile` from the test.

We only want to exclude derby here.

_p.s. I didn't see anything in http://slick.lightbend.com/doc/3.1.1/supported-databases.html need to be changed, since this fix won't add/remove any db feature._